index.html

```diff
@@ -1377,8 +1377,8 @@ <h2>Core Vocabulary Definitions</h2>
 <li><a href="#actionaffordance"><code>ActionAffordance</code></a></li>
 <li><a href="#eventaffordance"><code>EventAffordance</code></a></li></ul></section>
 <section><h3><code>PropertyAffordance</code></h3><p>An Interaction Affordance that exposes state of the
-          Thing. This state can then be retrieved (read) and/or
-          updated (write). Things can also choose to
+          Thing. This state can then be retrieved (read) and
+          optionally updated (write). Things can also choose to
```

Comment:
@egekorkan Did you mean to revert @danielpeintner's change from #1306? I'm guessing that was an accident?

Reply:
This is indeed weird and definitely unintentional. Should I revert #1306 and reopen and remerge it?

Reply:
I will take care of it in PR https://github.com/w3c/wot-thing-description/pull/1334